sample-new-architecture/src/Screens/GesturesTracingScreen.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import { Gesture, GestureDetector } from 'react-native-gesture-handler';
+import { getCurrentHub, Scope, traceGesture } from '@sentry/react-native';
+
+const GesturesTracingScreen = () => {
+  const gesture = Gesture.Pinch()
+    .onBegin(() => {
+      console.log('Pinch gesture begin');
+    })
+    .onEnd(() => {
+      console.log('Pinch gesture end');
+      startExampleSpan();
+    });
+
+  return (
+    <GestureDetector gesture={traceGesture('pinch', gesture)}>
```

Comment:
Is this the wrapper that people need to do in order to trace the gestures?
M: Maybe we call it `sentryTraceGesture or something like that? right now it's unclear if this is related to sentry instrumentation or not.

Reply:
I see, yes, that makes sense to me.